packages/material/scss/button/_variables.scss

```diff
@@ -254,4 +254,4 @@ $kendo-flat-button-focus-ring-opacity: null !default;
 
 /// The box-shadow transition of the Button.
 /// @group button
-$kendo-button-transition: box-shadow 280ms cubic-bezier( .4, 0, .2, 1 ) !default;
+$kendo-button-transition: box-shadow var(--kendo-motion-medium-3, 280ms) cubic-bezier( .4, 0, .2, 1 ) !default;
```

Comment:
I think _--kendo-motion-medium-2_ is closer to 280ms than _--kendo-motion-medium-3_ - could you please check if this change is intended?